docs/tutorials/sharded-computation.md

```diff
@@ -299,5 +347,12 @@ def layer_sharded(x, weights, bias):
 layer_sharded(x, weights, bias)
 ```
 
-This section has been a brief introduction of sharded and parallel computation;
-for more discussion of `shard_map`, see {doc}`../notebooks/shard_map`.
++++ {"id": "5c1d8146cd6f"}
+
+## Next steps
+
+This tutorial serves as a brief introduction of sharded and parallel computation in JAX.
+
+To learn about each SPMD method in-depth, check out these docs:
+- [Automatic parallelization with `jax.jit`](https://jax.readthedocs.io/en/latest/notebooks/Distributed_arrays_and_automatic_parallelization.html)
```

Comment:
Let's link using `{doc}` markup rather than HTTP hyperlinks, because this will cause a build failure if and when those docs are eventually moved.